client/src/main/java/io/pravega/client/security/auth/DelegationTokenProviderFactory.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.security.auth;
+
+import com.google.common.base.Preconditions;
+import io.pravega.client.segment.impl.Segment;
+import io.pravega.client.stream.impl.Controller;
+import io.pravega.common.Exceptions;
+
+/**
+ * Creates a {@link DelegationTokenProvider} instance.
```

Comment:
A proper Javadoc would be "Factory class for DelegationTokenProvider instances"

Reply:
Done.